Makefile

```diff
@@ -81,7 +92,8 @@ vet: ## Run go vet against code.
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" POD_NAMESPACE=default \
-		MESH_NAMESPACE=istio-system CONTROL_PLANE_NAME=istio-system go test $$(go list ./... | grep -v /e2e) -coverprofile cover.out
+		MESH_NAMESPACE=istio-system CONTROL_PLANE_NAME=istio-system go test -count=1 $$(go list ./... | grep -v /e2e) \
```

Comment:
If `-count=1` (as well as changes to code coverage) is important, that should be another PR explaining why :) Otherwise, it will be just lost under the cover of the innocent dep upgrade commit :)

Reply:
https://github.com/opendatahub-io/odh-model-controller/pull/449